### ISSUE TITLE
val_rpn_bbox_loss decreases but plateaus around 1.1, while the other Mask RCNN losses keep decreasing

### ISSUE THREAD

**Issue by cpoptic** [matterport/Mask_RCNN#1812]
I'm training a modified version of MaskRCNN on ~1700 images of dimension 512x512.
LR = 0.0003.

I'm also using image augmentation via imgaug with LR flipping 0.5 of the time and random scaling and translation horizontally and vertically from [0.9 1.1], along with some very small rotations of +/- 5 degrees.

val_rpn_bbox_loss decreases but plateaus around 1.1, while the other Mask RCNN losses keep decreasing

Due to the smaller size, I added a few dimensions to the Anchors to capture objects are more scales.
I still kept the standard Anchor Ratio of [1/2, 1, 2], although some of the objects I'm trying to classify are little wider than can fit tightly bounded inside a 1:2 anchor box, so I might try adding [1/3, 3] to the default ratios.

I'm tracking the training with TensorBoard, and the training loss is dropping with each epoch nicely.
Also the other 4 losses are decreasingly nicely with each epoch, up through around 50 or so epochs:

val_rpn_class_loss 
val_mrcnn_class_loss 
val_mrcnn_bbox_loss 
val_mrcnn_mask_loss 

but the pesky val_rpn_bbox_loss is remaining stubbornly high and won't decrease.

What could be causing this?

Also, I've run the model with both image augmentation turned on as well as off, and this doesn't appear to affect the val_rpn_bbox_loss.

I have a screenshot of the losses in Tensorboard, but despite several attempts, GIthub won't let me upload an image to this issue.

**Comment by FTRobles** [comment]
Hi, did you solve the problem? I am facing the same issue.
